controllers/productController.js

```diff
@@ -6,11 +6,19 @@ const getProducts = catchAsync(async (req, res) => {
     const limit = req.query.limit;
     const gender = req.query.gender;
 
-    const products = await productService.getProducts(offset,limit,gender);
+    const products = await productService.getProducts(offset, limit, gender);
     
     res.status(200).json({ data : products });
 });
 
+const getProductDetails = catchAsync(async (req, res) => {
+    let productId = parseInt(req.params.productId);
```

Comment:
`productId`는 추후 재할당되지 않는 변수이기 때문에 `const` 키워드를 사용하는 것이 좋을 것 같습니다!
```suggestion
    const productId = parseInt(req.params.productId);
```